charts/cronjob/values.yaml

```diff
@@ -2,6 +2,7 @@ image:
   repository: 'commercetools/commercetools-email-retry-processor'
   tag: '1.0.0'
   pullPolicy: 'IfNotPresent'
+  command: ['cmd1', 'cmd2']
```

Comment:
What about put there a `hello world` cmd?

Reply:
don't you think `'cmd1', 'cmd2'` are expressive enough?

Reply:
They just don't work, but other than that, it's ok.

Reply:
Agreed. @svenmueller mentioned before that 
> charts should be able to run out-of-the-box with default values.

 So in this case, I left it out empty and added a comment to describe it next to it. WDYT @lojzatran ?

Reply:
Ok to me, leave it empty then. 👍